scripts/release.py

```diff
@@ -42,6 +45,7 @@
     PRINT - Whether or not the release notes should be printed to CLI or be used to create a Github release. Default is 0 e.g. PRINT=1 or PRINT=0
```

Comment:
Could we also remove the `PRINT `option then?

Reply:
Sure!